tests/acceptance/addon-dummy-generate-test.js

```diff
@@ -240,4 +240,69 @@ describe('Acceptance: ember generate in-addon-dummy', function () {
     await generateInAddon(['server', '--dummy']);
     expect(file('server/index.js')).to.exist;
   });
+
+  // [WIP] ember addon --lang flag
+  // -------------------------------
+  // Good: Default
+  it('ember addon without --lang flag (default) has no lang attribute in dummy index.html', async function () {
+    await ember(['addon', 'foo', '--skip-npm', '--skip-bower', '--skip-git']);
+    expect(file('tests/dummy/app/index.html')).to.contain('<html>');
+  });
```

Comment:
I'd prefer to account for this in the main blueprinting tests for default output of `new`/`addon` output here:

https://github.com/ember-cli/ember-cli/blob/5237dc7f67e766c8da1a56e329b6c91369c0c34c/tests/acceptance/new-test.js#L439-L460

Reply:
Moved in 29e0a4b1 👍